src/application.js

```diff
@@ -8,11 +8,8 @@ import Region           from './region';
 var Application = MarionetteObject.extend({
   cidPrefix: 'mna',
 
-  constructor: function(options) {
-    options = options || {};
-
+  constructor: function(options = {}) {
     this._initRegion(options);
```

Comment:
hmm this is actually broken.
this shouldn't pass options to `_initRegion`
It should
```
  // set options temporarily so `_initRegion` can use `getOption`
  this.options = options;
```

Concurrent branch issues here.. also Application isn't tested super well.. (but it will be)